### ISSUE TITLE
Taxon constraint: GO:0019853    L-ascorbic acid biosynthetic process 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28890]
Please provide as much information as you can: 

* **GO term ID label:**
taxon restrict GO:0019853    L-ascorbic acid biosynthetic process  for yeast to block

UniProtKB:P38235 | YBR053C | involved_in | GO:0019853    L-ascorbic acid biosynthetic process | ECO:0000318   IBA | GO_REF:0000033 | MGI:MGI:108024 more... | 559292 Saccharomyces cerevisiae (strain ATCC 204508 / S288c)

I'm surprised that mice can make vitamin C?  I wanted to check it but the page in the xref is giving an error @LiNiMGI  do you know why?

* **Request to add a taxon constraint:**
* * **only in taxon: **
* * **never in taxon: **

* **Request to remove a taxon constraint:**
Please specify

* **Supporting evidence if available (e.g PMID):**

**Comment by LiNiMGI** [comment]
@ValWood Hi Val, I am not sure what you meant by "the page in the xref is giving an error", in MGI, the reference for the annotation is PMID:16585534 (https://pubmed.ncbi.nlm.nih.gov/16585534/). thanks!  Li

**Comment by ValWood** [comment]
Hi @LiNiMGI 
go to
https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=P38235&aspect=molecular_function
and follow the links to MGI from the "with" field
(I think it happens for quite a lot of the MGI links from "with fields", I have seen it a few times).

I wonder if the Alliance bug  that I reported retrieving GO data from MGI is related to this? I can't see how, but it seems odd to have 2 GO-related bugs that only affect MGI ?

**Comment by ValWood** [comment]
Actually, I see what's causing this; I'm sure I reported this way back. Is it because of the MGI:MIGI:prefix?
Could this be causing the Alliance problems too?

**Comment by LiNiMGI** [comment]
@ValWood 
Hi Val, I don't see this problem in alliance page, could this be a quick GO issuer?
https://www.alliancegenome.org/gene/SGD:S000000257

**Comment by LiNiMGI** [comment]
@ValWood 
MGI:MGI:108024 
Quick GO link to: https://www.informatics.jax.org/accession/MGI
Alliance link to: https://www.informatics.jax.org/marker/MGI:108024

**Comment by ValWood** [comment]
The Alalince issue is probably unrelated, its that MGI GO EXP annotations are not retrieved suing the ortholog feature. We can ignore that. maybe this is a  QuickGO issue.

**Comment by ValWood** [comment]
There is definitely a problem accessing MGI from the QuickGO "with" field.

**Comment by ValWood** [comment]
Actions from this ticket
Add  
GO:0019853    L-ascorbic acid biosynthetic process 
never in yeast
never in human

**Comment by deustp01** [comment]
> * vitamin C is not synthesized in human

Also not in guinea pigs. Don't know about chimpanzees and gorillas. I think (too overloaded to research properly) that for humans, a single enzyme step in a synthetic pathway that is well-conserved in mammals, so one could look at a sequence alignment of that enzyme sequence over mammalian taxa to figure out how broad the "never in taxon" constraint should be.
